spec.bs

```diff
@@ -4755,12 +4762,30 @@ dictionary ScoringBrowserSignals {
   required unsigned long biddingDurationMsec;
   required DOMString bidCurrency;
 
+  record<DOMString, DOMString> renderSize;
   unsigned long dataVersion;
   sequence<USVString> adComponents;
   boolean forDebuggingOnlyInCooldownOrLockout = false;
 };
 </xmp>
 
+<div algorithm>
+  To <dfn>convert an ad size to a map</dfn> given an [=ad size=] |adSize|:
+  
+  1. Let |dict| be a new empty [=map=].
+  1. Let |widthStr| be a new empty [=string=].
+  1. Let |jsWidth| be |adSize|'s [=ad size/width=], [=converted to an ECMAScript value=].
+  1. Append [$ToString$](|jsWidth|) to |widthStr|.
+  1. Append |adSize|'s [=ad size/width units=] to |widthStr|.
+  1. [=map/Set=] |dict|["width"] to |widthStr|.

```

Comment:
There's no append method for string.
Probably can do:
`[=map/Set=] |dict|["width"] to the result of [=string/concatenating=] «[$ToString$](|jsWidth|),  |adSize|'s [=ad size/width units=]».`